server/README.md

```diff
@@ -168,15 +166,15 @@ We use [helm](https://helm.sh/docs/) to manage our deployments. Helm allows us t
 
 1. on your local machine, open Visual Studio Code in the `firmware/boron-ins-fsm` directory and open the Particle Workbench extension
 
-1. run `git checkout production && git pull origin production` to get the latest version of the the code
+1. if you haven't already, run `git checkout production && git pull origin production` to get the latest version of the the code
 
 1. in `BraveSensorProductionFirmware.ino`, change `BRAVE_PRODUCT_ID` to the ID of the "Production Sensor Devices" project (`15479`) [NOTE: do NOT commit this change]
 
-1. click Target --> "Configure for device" and type `3.3.1` to choose the OS and then `boron` to choose the device
+1. if you haven't already, click Target --> "Configure for device" and type `3.3.1` to choose the OS and then `boron` to choose the device
 
-1. click Compile --> "Local Compile" (this step can take a few minutes the first time; you can work on the next step in parallel while you wait for this)
+1. if you haven't already, click Compile --> "Local Compile" (this step can take a few minutes the first time; you can work on the next step in parallel while you wait for this)
 
-1. copy the generated `/target/3.3.1/boron/boron-ins-fsm.bin` file somewhere to keep for reference. Rename to `v<version number>_production.bin` (for example `v50100_production.bin`)
+1. copy the generated `/target/3.3.1/boron/boron-ins-fsm.bin` file to [Google Drive](https://drive.google.com/drive/u/0/folders/1QVvBvGM3MP9VU5-8AVG3Nf0KddeE5_uz) for future reference. Rename to `v<version number>_production.bin` (for example `v50100_production.bin`)
 
 1. in your browser navigate to Particle Console --> Brave Technology Coop --> Production Sensor Devices --> Firmware
 
```

Comment:
I can't add comments to those lines, but I would also remove periods after lines 217, 221, and 453 for consistency (Also because I think we almost got thrown off by the period haha)